dali/python/bundle-wheel.sh

```diff
@@ -48,7 +48,15 @@ STRIP_DEBUG=${2:-NO}
 TEST_BUNDLED_LIBS=${3:-NO}
 OUTWHLNAME=${4:-$(basename $INWHL)}
 DEPS_PATH=${5:-/usr/local}
-OUTDIR=/wheelhouse
+OUTDIR=${6:-/wheelhouse}
```

Comment:
Can you adjust build_helper.sh, build.sh, Dockerfile, and Dockerfile.build.aarch64-linux to allow passing these variables from the outside so even the local build using docker environment could benefit from it?

Reply:
I tried adding it, compression might be useful, the target directory for docker would require handling the in-docker and out of docker targets probably, so I just left it as is.

Reply:
Looks good now.